artifacts.md

```diff
@@ -6,7 +6,11 @@ description: Encapsulating data in Aqueduct
 
 _Artifacts_ are thin wrappers around data objects in Aqueduct. Wrapping data in an Artifact allows Aqueduct to track that artifact across workflow runs help you understand how your data is changing over time.&#x20;
 
-Aqueduct currently supports `TableArtifacts`, which are encapsulations of tabular or relational data, and [`ParameterArtifacts`](workflows/parameterizing-a-workflow.md), which wrap mutable json values. We plan to add support for other data types in the future.
+Aqueduct currently supports the following types of _Artifacts_:
+* `TableArtifacts`, which are encapsulations of tabular or relational data
+* `BoolArtifacts`, which are encapsulations of boolean data
+* `NumericArtifacts`, which are encapsulations of numeric data
+* `GenericArtifacts`, which are all other data type encapsulations that are not covered above. For supported data types, see [Artifact Data Types](artifacts/artifact-data-types.md)
```

Comment:
This might be confusing. Rather than listing the class types, maybe we could just copy over the data types documentation here? What do you think?